### PR TITLE
Validation

### PR DESCRIPTION
**Pull request by rpire**
## Changes made:

- Implemented a function that validates if email input is all in lower case.
- Added form-validation.js.
- Added a new `small` tag in which the error message will be displayed.
- Applied styling to the error message.

### CODE REVIEW

File: form-validation.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      form.addEventListener('submit', (event) => {
+        event.preventDefault();
+      });
+      message.innerHTML = '* Make sure your email is in lower case.';
+    } else {
+      form.submit();
+    }
+  }
+}
+validation();
```
Comment (Dipeshtwis) [inline form-validation.js:18]: - [ ] For now, you have mentioned to submit the form if the email is in the correct form and follow the requirement but there is an issue with this, if I just enter the email correctly without filling any other input field and trying to submit, it is submitting which is not a good behaviour of form. Please fix this. :slightly_smiling_face:

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @rpire, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: The contact form contains validation checking if the email is written in all lowercase.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rpire** [comment]
Hello @Dipeshtwis 🖐

Thank you for reviewing my work and noticing my big error! 😅

I will make the requested changes and ask for another review as soon as I have it all ready.

Thanks again, Cheers and Happy coding! 🚀🚀

File: form-validation.js (hunk 1 above)
Comment (rpire) [reply form-validation.js]: Requested change implemented 🤓

**Review by Shaher-11 — APPROVED** [review]
#### Hello @rpire,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle::muscle:

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Comment by rpire** [comment]
## Thank you! 🙌🙌

I will proceed to merge it. 🤓🚀

**Pull request merged**